app/javascript/lib/scp-api.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * @fileoverview JavaScript client for Single Cell Portal REST API
+ *
+ * Succinct, well-documented SCP API wrappers, also enabling easy mocks
+ *
+ * API docs: https://singlecell.broadinstitute.org/single_cell/api
+ */
+import camelcaseKeys from 'camelcase-keys';
+
+const defaultBasePath = '/single_cell/api/v1';
+
+let defaultInit = {
+  method: 'GET',
+  headers: {
+    'Content-Type': 'application/json',
+    'Accept': 'application/json'
+  }
+}
+
+/**
+ * Get a one-time authorization code for download, and its lifetime in seconds
+ *
+ * TODO:
```

Comment:
I don't know what the team norms are around TODOs in code.  In general, I lean towards 'all TODOs must reference a jira ticket', otherwise replace the TODO with to 'maybe' or 'perhaps' or 'would be nice to...', but I'm open to other conventions

Reply:
Addressed in 034ff9d0.  SCP-2144 tracks linting for such issues.